### PR TITLE
Update pymemcache instrumentation to follow semantic conventions

### PR DESCRIPTION
**Pull request by lonewolf3739**
# Description
Part of  #159 

# Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] `tox -e test-instrumentation-pymemcache`

### CODE REVIEW

**Comment by lonewolf3739** [comment]
cc @lzchen

File: instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py [hunk 1]
```diff
@@ -115,7 +114,7 @@ def wrapper(wrapped, instance, args, kwargs):
 @_with_tracer_wrapper
 def _wrap_cmd(tracer, cmd, wrapped, instance, args, kwargs):
     with tracer.start_as_current_span(
-        _CMD, kind=SpanKind.INTERNAL, attributes={}
+        cmd, kind=SpanKind.CLIENT, attributes={}
     ) as span:
```
Comment (lzchen) [inline instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py:118]: From before, does pymemcache not have the concept of `name`?

Comment (lonewolf3739) [reply instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py:118]: I am not sure if I understood you correctly, but memcache supports this list of [commands](https://github.com/memcached/memcached/wiki/Commands). We are using the command as a span name.

Comment (lzchen) [reply instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py:118]: Sorry I was referring to adding the `db.name` attribute.

Comment (lonewolf3739) [reply instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py:118]: No, it doesn't have any concept of databases that can be used for `db.name`.

**Review by lzchen — APPROVED** [review]

**Pull request merged**